week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -26,8 +34,19 @@ function exerciseOne(arrayOfPeople) {
  */
 function exerciseTwo(shopping) {
   //Write your code in here
+
+  let getContent = document.getElementById("content");
+  let createList = document.createElement("ul");
+  getContent.appendChild(createList)
+  for (let i = 0; i<shopping.length; i++){
+  let createEl = document.createElement("li");
+  createEl.innerHTML = shopping[i];
```

Comment:
Considering `shopping[i]` is only a string, is `innerHTML` necessary here or could it be replace with one of the other DOM methods?

Reply:
Hi Antagoni, thanks for the comment. I think textContext works better here.